recipes/speedb/all/conanfile.py

```diff
@@ -0,0 +1,138 @@
+        if self.options.with_snappy:
+            self.requires("snappy/1.1.10")
+        if self.options.with_lz4:
+            self.requires("lz4/1.9.4")
+        if self.options.with_zlib:
+            self.requires("zlib/[>=1.2.11 <2]")
+        if self.options.with_zstd:
+            self.requires("zstd/1.5.5")
+        if self.options.with_core_tools:
+            self.requires("gflags/2.2.2")
+            self.requires("readline/8.2")
+
+    def validate(self):
+        if self.settings.compiler.cppstd:
+            check_min_cppstd(self, self._min_cppstd)
+        minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
+        if minimum_version and Version(self.settings.compiler.version) < minimum_version:
+            raise ConanInvalidConfiguration(
+                f"{self.ref} requires C++{self._min_cppstd}, which your compiler does not support."
+            )
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["WITH_JEMALLOC"] = False
+        tc.variables["WITH_LIBURING"] = False
+        tc.variables["WITH_SNAPPY"] = self.options.with_snappy
+        tc.variables["WITH_LZ4"] = self.options.with_lz4
+        tc.variables["WITH_ZLIB"] = self.options.with_zlib
+        tc.variables["WITH_ZSTD"] = self.options.with_zstd
+        tc.variables["WITH_TESTS"] = False
+        tc.variables["WITH_BENCHMARK_TOOLS"] = False
+        tc.variables["WITH_CORE_TOOLS"] = self.options.with_core_tools
+        tc.variables["WITH_GFLAGS"] = self.options.with_core_tools
+        tc.variables["WITH_TOOLS"] = False
+        tc.variables["ROCKSDB_BUILD_SHARED"] = self.options.shared
+        tc.generate()
+        deps = CMakeDeps(self)
+        deps.generate()
+
+    def _patch_sources(self):
+        apply_conandata_patches(self)
+        # to avoid wrong CPU detection, modify not to use "-march=native" in apple-clang
+        if cross_building(self) and self.settings.compiler == "apple-clang":
+            replace_in_file(self,
+                os.path.join(self.source_folder, "CMakeLists.txt"),
+                """elseif(NOT CMAKE_SYSTEM_PROCESSOR MATCHES "^(powerpc|ppc)64" AND NOT HAS_ARMV8_CRC)""",
+                "elseif(FALSE)"
+            )
+
+    def build(self):
+        self._patch_sources()
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def package(self):
+        copy(self, pattern="LICENSE", dst=os.path.join(self.package_folder, "licenses"), src=self.source_folder)
+        cmake = CMake(self)
+        cmake.install()
+
+        rmdir(self, os.path.join(self.package_folder, "lib", "cmake"))
+        rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
+
+    def package_info(self):
+        self.cpp_info.libs = ["speedb"]
+        self.cpp_info.set_property("pkg_config_name", "speedb")
```

Comment:
```suggestion
        self.cpp_info.set_property("pkg_config_name", "speedb")
        self.cpp_info.set_property("cmake_file_name", "Speedb")
        self.cpp_info.set_property("cmake_target_name", "Speedb::speedb")
```

https://github.com/speedb-io/speedb/blob/main/CMakeLists.txt#L1287
https://github.com/speedb-io/speedb/blob/main/CMakeLists.txt#L1337